docs/examples/napari/napari_img_math.md

```diff
@@ -201,12 +201,12 @@ of your choice.  Then, whenever the gui widget *or the original function* are
 called, the result will be passed to your callback function:
 
 ```python
-def print_mean(event):
+@image_arithmetic.called.connect
+def print_mean(value):
     """Callback function that accepts an event"""
-    # the event.value attribute has the result of calling the function
-    print(event.value.mean())
+    # the value attribute has the result of calling the function
+    print(value.mean())
```

Comment:
```suggestion
    print(np.mean(value))
```

Unrelated to this PR but since `mean` is not part of the [emerging Array API standard](https://data-apis.org/array-api/latest/API_specification/array_object.html), might as well get a head start on this. 😂